dev/emmy/sci_extensions.cljs

```diff
@@ -0,0 +1,10 @@
+(ns emmy.sci-extensions
```

Comment:
we don't need this yet, since none of the viewers are actually using Emmy code on the client side.